api/queries_issue.go

```diff
@@ -608,6 +609,129 @@ func IssueUpdate(client *Client, repo ghrepo.Interface, params githubv4.UpdateIs
 	return err
 }
 
+func IssueTransfer(client *Client, destRepo *Repository, issue Issue) (string, error) {
+	mutation := `
+	mutation transferIssue($input:TransferIssueInput!){
+		transferIssue(input: $input){
+			issue {
+				url
+			}
+		}
+	}
+	`
+
+	type response struct {
+		TransferIssue struct {
+			Issue struct {
+				URL string
+			}
+		}
+	}
+
+	variables := map[string]interface{}{
+		"input": map[string]interface{}{
+			"issueId":      issue.ID,
+			"repositoryId": destRepo.ID,
+		},
+	}
+
+	var resp response
+
+	err := client.GraphQL(destRepo.RepoHost(), mutation, variables, &resp)
+	if err != nil {
+		return "", err
+	}
+	return resp.TransferIssue.Issue.URL, nil
+}
+
+// DestinationReposToTransferIssue fetches all the eligible repos to which the viewer can transfer issues
+func DestinationReposToTransferIssue(client *Client, srcRepo ghrepo.Interface) ([]*Repository, error) {
```

Comment:
Awesome work, but we can get rid of this whole function, per my previous comment.